resources/views/account/partials/orders.blade.php

```diff
@@ -1,20 +1,45 @@
-<template v-if="data.customer.orders.items.length">
-    <x-rapidez-ct::sections>
-        <x-rapidez-ct::card.inactive class="flex-col" v-for="order in data.customer.orders.items">
-            <x-rapidez-ct::title.lg>
-                @lang('Order') #@{{ order.number }}
-            </x-rapidez-ct::title.lg>
-            <a :href="'/account/order/'+order.number">
-
+<div>
+    <template v-if="data.customer.orders.items.length">
+        <x-rapidez-ct::sections>
+            <a
+                class="block"
+                v-for="order in data.customer.orders.items"
+                :href="'/account/order/' + order.number"
+            >
+                <x-rapidez-ct::card.inactive class="flex-col">
+                    <x-rapidez-ct::title.lg>
+                        @lang('Order') #@{{ order.number }}
+                    </x-rapidez-ct::title.lg>
+                    <x-rapidez-ct::card.white class="mt-5 flex flex-wrap items-center gap-x-3 md:gap-x-8">
+                        <x-heroicon-s-shopping-cart class="h-5 text-ct-inactive" />
```

Comment:
sometimes i see the solid version of heroicon and sometimes i see the outline version, is there a specific reason for this?

Reply:
usually we use outline versions but you preffered this version when you were sitting next to me